Simulator/Test/test_conversions.py

```diff
@@ -89,7 +140,13 @@ def test_skew():
     np.testing.assert_allclose(zero, conv.skew(x)@x, atol=tol)
 
 def test_mjd_2_GMST():
```

Comment:
This test is causing an error. The tolerance pytest pegs it at is ~10E-3 which is much higher that atol. Looks like you notes it would be an inaccurate test though. Any ideas why and how to make it better?

Reply:
Yes so, this test seems pretty important, but I can't find a good way to implement it. The current test is just based on online calculators, which give similar (but not identical) answers to our function. I left the buggy test there so it wouldn't just get forgotten.

Reply:
Yeah, we definitely need the test, but I wonder if there's a more accurate way to do it, as you noted. There's a library called `julian` that does julian-date conversions that could have what we're looking for as a means of comparison.

Reply:
I took a look at the `julian` library. Unfortunately, it only converts between JD/MJD to UTC (in a datetime object). However, in order to make the jump from UTC to GMST, we need to use those nasty polynomials that approximate it.

Sadly, after googling this conversion online for a while, it seems that there are several variations of this conversion from GMST <-> UTC. I had previously implemented the conversion given at [this source from Harvard](https://www.cfa.harvard.edu/~jzhao/times.html), but then found that it was incompatible with what Paul/Ethan had coded in their functions. We then switched to the conversion provided in the Vallado textbook.

Reply:
I'm confused how there can be multiple, incompatible, conversions between two objective units... Did you look into the discrepancy at the time?